src/compass/version.py

```diff
@@ -8,6 +8,7 @@
 # release history
 Tag = collections.namedtuple('Tag', 'version date')
 release_history = (
+    Tag('0.5.2', '2023-10-04'),
```

Comment:
Not sure if you have seen the email from S. Lewis. He asked use to tag the COMPASS repository for our previous point release. Not sure how to go about this and if it is possible to tag historical states of the repository. May be for safety we should use `0.5.3`?

Reply:
@vbrancat I think it's doable to tag the current main branch to `0.5.2` with the small PR for that. Do we need to deliver the Docker image to SDS after tagging it?

Reply:
Submitted another PR #213 to bump the version. #213 need to be merged before this PR to make the change to take place in order.